src/zenml/client.py

```diff
@@ -2386,6 +2392,113 @@ def _delete_artifact_metadata(
         self.zen_store.delete_artifact(artifact.id)
         logger.info(f"Deleted metadata of artifact '{artifact.uri}'.")
 
+    # ----------------
+    # - Run Metadata -
+    # ----------------
+
+    def create_run_metadata(
```

Comment:
This is on the client so that it can potentially be called manually within pipeline steps? How would that be done? Do we have a way to acces the current pipeline ID in the step at runtime? We'll definitely need docs around this.

Reply:
Yeah it's in the client so users can log custom stuff if they really want/need to. I think you could get both the pipeline run ID and step run ID via `StepContext`. But I haven't experimented with that too much yet. Not even sure whether we should let users know this functionality exists.